src/tss2-esys/esys_crypto_ossl.c

```diff
@@ -1121,6 +1121,8 @@ iesys_cryptossl_sym_aes_decrypt(uint8_t * key,
  */
 TSS2_RC
 iesys_cryptossl_init() {
+    ENGINE_load_builtin_engines();
     OpenSSL_add_all_algorithms();
     return TSS2_RC_SUCCESS;
 }
+
```

Comment:
Please don't add unneeded extra empty lines like this.

Reply:
Sorry about that. Not intentional.